syntax = "proto3";
package Msg2Proxy;

import "BaseDefine.proto";

message LoginReq {
    uint32 session_id = 1;
    uint32 user_id = 2;
    string password = 3;
    BaseDefine.ClientType client_type = 4;
}

message LoginRsp {
    BaseDefine.ResultType result_code = 1;
    uint32 session_id = 2;
    BaseDefine.UserInfo user_info = 3;
    string result_string = 4;
}